#89: Implement setProperties request. (#91)
exasol/row-level-security-lua: doc/user_guide/user_guide.md

Spaces around the table names are ignored.

### Changing the Properties of an Existing Virtual Schema

While you could in theory drop and re-create an Virtual Schema, there is a more convenient way to apply changes in the adapter properties.

> Use `ALTER VIRTUAL SCHEMA ... SET` to update an existing Virtual Schema to update the properties and with them an existing Virtual Schema.

Reviewer: ```suggestion
Use `ALTER VIRTUAL SCHEMA ... SET ...` to update the properties of an existing Virtual Schema.
```

Example:

```sql
ALTER VIRTUAL SCHEMA RLS_VIRTUAL_SCHEMA
SET SCHEMA_NAME = '<new schema name>'
```

> You can for example change the `SCHEMA_NAME` property to point the Virtual Schema to a new source schema.

Reviewer: ... or add a table filter?
